### ISSUE TITLE
feature request: Automatic lookup GraalVM installation directory

### ISSUE THREAD

**Issue by yujiorama** [mike-neck/graalvm-native-image-plugin#24]
## Summary

Add extension property so that to be able to lookup install directory of GraalVM.

## Detail

There are some strategy to lookup installation directory.

* Find `gu` commnd from environment variable `PATH`.
* Find `gu` commnd from environment variables `SDKMAN_CANDIDATES_DIR` recursively.
* Find `gu` commnd from environment variables `JAVA_HOME` recursively.
* Find `gu` commnd from system property `java.home` recursively.

Some configuration example.

```
nativeImage {
  findGraalVmHome = true
  mainClass = 'com.example.App'
  executableName = 'my-native-application'
  arguments(
      '--no-fallback',
      '--enable-all-security-services',
      '--initialize-at-run-time=com.example.runtime',
      '--report-unsupported-elements-at-runtime'
  )
}
```

**Comment by mike-neck** [comment]
What is the usecase of this feature?

**Comment by yujiorama** [comment]
I just realized earlier that this was a problem that could be solved by the env feature in sdkman.
This request is canceled.

In addition, the request was intended to be used in the following situations.

The OS is Windows 10.
We were involved in a project that required a specific version of the JDK, so we needed to manage multiple versions of the JDK.

To manage multiple versions of the JDK, I was using scoop, the package manager, rather than sdkman.

scoop will add the bin directory of the installed JDK to the PATH environment variable.
Therefore, we thought that an automatic search mechanism would make it easier to avoid setting up each project.
